site_scons/prereq_tools/base.py

```diff
@@ -1470,24 +1470,26 @@ def has_missing_system_deps(self, env):
     def parse_config(self, env, opts):
         """Parse a pkg-config file"""
         if self.pkgconfig is None:
-            return False
-
+            return
         path = os.environ.get("PKG_CONFIG_PATH", None)
         if path and "PKG_CONFIG_PATH" not in env["ENV"]:
             env["ENV"]["PKG_CONFIG_PATH"] = path
-        if self.component_prefix:
+        if not self.use_installed and not self.component_prefix == "/usr":
+            path_found = False
             for path in ["lib", "lib64"]:
                 config = os.path.join(self.component_prefix, path, "pkgconfig")
                 if not os.path.exists(config):
                     continue
                 env.AppendENVPath("PKG_CONFIG_PATH", config)
+            if not path_found:
+                return
```

Comment:
Continued from Ashley's comment, this would throw an exception if `path_found` is not defined

Reply:
Ah, this is under the same `if` scope. Please ignore this comment. But I think Ashley's comment above is still valid.